### DISCUSSION TITLE
bridge xterio chain to bnb

### DISCUSSION THREAD

**Discussion by vanty225339** [blockscout/blockscout#9992] [Support]
Confirmation was successful but I didn't receive my money
In challenge period (none)
Ready for relay (none)
Relayed (none)
https://xterscan.io/tx/0x746b6ed1d038067a6e0fdb17c5b0f375bb3a05c64392c4cffb318cb5c479ca45

**Comment by Alex93799** [comment]
У меня такая же проблема

**Comment by litaining** [comment]
Has the problem been resolved? Friend, I am facing the same problem as you.

**Comment by vanty225339** [comment]
No, they didn't give me a gmail or anything to contact me!
All contact information is not available

Vào Th 4, 8 thg 5, 2024 vào lúc 08:45 litaining ***@***.***>
đã viết:

**Comment by vanty225339** [comment]
I got my money I know how!
0xe9cE7e0a5eFFbD68EFcFa7B0E00bbb5c4A29B452

Vào Th 5, 9 thg 5, 2024 vào lúc 20:07 Ty Le ***@***.***> đã
viết:

**Comment by vanty225339** [comment]
https://bnb.xterscan.io/tx/0x746b6ed1d038067a6e0fdb17c5b0f375bb3a05c64392c4cffb318cb5c479ca45

Vào Th 2, 13 thg 5, 2024 vào lúc 21:23 Ty Le ***@***.***> đã
viết:

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
